README.md

```diff
@@ -120,6 +121,44 @@ $ hypervisor
 
 You can open up the hypervisor UI on `localhost:8000`. 
 
+### Windows
+
+Skywire node may be run on Windows, but this process requires some manual operations at the moment.
+
+In order to run the skywire visor on Windows, you will need to manually build it. Do not forget the `.exe` extensions.
+
+`go build -o ./skywire-visor.exe ./cmd/skywire-visor`
+
+Apps may be built the same way:
+
+`go build -o ./apps/vpn-client.exe ./cmd/apps/vpn-client`
+
+Apps should be stated in the config without `.exe` extension, as usual.
+
+Some log lines may seem strange due to the Windows encoding in terminal. To make life easier you may use:
+```
+REM change CHCP to UTF-8
```

Comment:
Is this comment necessary?

Reply:
I think it is alright to mention that as otherwise the output is hard to read or use to debug.

Reply:
@jdknives I think this could be added to line 138 as it's a part of the description. No need to execute a comment.